src/ScottPlot5/ScottPlot5 Demos/Avalonia Demo/Views/MainWindow.axaml

```diff
@@ -0,0 +1,80 @@
+<Window xmlns="https://github.com/avaloniaui"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:vm="using:Avalonia_Demo.ViewModels"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	    xmlns:local="clr-namespace:Avalonia_Demo"
+        mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
+        x:Class="Avalonia_Demo.Views.MainWindow"
+        x:DataType="vm:MainWindowViewModel"
+        Icon="/Assets/avalonia-logo.ico"
+		Background="#9a4993"
+        WindowStartupLocation="CenterScreen"
+        Title="ScottPlot Avalonia Demo" Height="700" Width="500"
+		>
+
+	<Design.DataContext>
+		<!-- This only sets the DataContext for the previewer in an IDE,
+             to set the actual DataContext for runtime, set the DataContext property in code (look at App.axaml.cs) -->
+		<vm:MainWindowViewModel/>
+	</Design.DataContext>
+
+	<DockPanel>
+		<DockPanel DockPanel.Dock="Top" Background="#67217a">
+			<Image Width="48" Margin="10 0 5 0" Name="LogoImage"
+				   Source="/Assets/logo-128.png"
+			/>
+			<StackPanel Margin="5">
+				<Label
+                    Content="ScottPlot Avalonia Demo"
+                    Foreground="WhiteSmoke"
+                    FontSize="26"
+                    FontWeight="SemiBold"
+                    Padding="0 5 0 0"
+                    Margin="0"/>
+				<Label
+                    Name="Subtitle"
+                    Content="{Binding VersionStringFull}"
```

Comment:
could it be space or tab only,
maybe you can use `xamlstyler.console` to format this axaml file while in dotnet cli,
you can also install the vsix in vs 2022
and also ,you can find ext in vscode name:` dabbinavo.xamlstyler`

Reply:
I went with `xstyler -d "path"`. It's a shame that this can't be configured with a `.editorconfig` file like our C# code.